jfrog-cli/artifactory/utils/golang/project/project.go

```diff
@@ -346,7 +346,7 @@ func (project *goProject) archiveProject(version string) (string, error) {
 
 // Parse module name from go.mod content.
 func parseModuleName(modContent string) (string, error) {
-	r, err := regexp.Compile(`module ([\w\.@:%_\+-.~#?&]+/?.+)`)
+	r, err := regexp.Compile(`module "?([\w\.@:%_\+-.~#?&]+/?.+\w)`)
```

Comment:
This reg exp is being compiled multiple times throughout the execution flow. The compilation of a reg exp is an expensive operation.
We should probably compile it only once, do it lazily and put it in a util, because it is the same for all projects.